rotkehlchen/tests/data_migrations/test_migrations.py

```diff
@@ -1,29 +1,19 @@
+from unittest.mock import patch
+
 import pytest
 
-from rotkehlchen.constants.assets import A_ETH, A_BTC
-from rotkehlchen.data_migrations.manager import DataMigrationManager
+from rotkehlchen.constants.assets import A_BTC, A_ETH
+from rotkehlchen.data_migrations.manager import DataMigrationManager, MigrationRecord
 from rotkehlchen.exchanges.data_structures import Trade
 from rotkehlchen.fval import FVal
 from rotkehlchen.tests.utils.exchanges import check_saved_events_for_exchange
-from rotkehlchen.typing import (
-    Location,
-    AssetAmount,
-    Fee,
-    Price,
-    Timestamp,
-    TradeType,
-)
+from rotkehlchen.typing import AssetAmount, Fee, Location, Price, Timestamp, TradeType
 
 
 @pytest.mark.parametrize('use_custom_database', ['data_migration_v0.db'])
-@pytest.mark.parametrize('data_migration_version', [0])
-def test_migration_1(
-    use_custom_database,  # pylint: disable=unused-argument
```

Comment:
@yabirgb you do not need to be adding the fixtures in the test again if they are already pulled by a fixture you have added. In this case the `rotkehlchen_api_server`

Reply:
With the fixtures I had a lot of trouble. Thanks